package.json

```diff
@@ -43,11 +42,7 @@
   "devDependencies": {
     "blip-mock-data": "tidepool-org/blip-mock-data#v0.3.7",
     "chai": "2.1.2",
-    "gulp": "3.8.11",
-    "gulp-jshint": "1.9.4",
-    "gulp-react": "3.0.1",
-    "jest-cli": "0.4.0",
-    "jshint-stylish": "1.0.1",
+    "jsxhint": "^0.15.1",
```

Comment:
nit-pic - we are using strict versions everywhere so this should be  ``"jsxhint": "0.15.1"``